code/modules/mob/living/simple_animal/friendly/mouse.dm

```diff
@@ -158,6 +158,19 @@
 	// Tom fears no cable.
 	ADD_TRAIT(src, TRAIT_SHOCKIMMUNE, SPECIES_TRAIT)
 
+/mob/living/simple_animal/mouse/white/Brain
+	name = "Brain"
+	real_name = "Brain"
+	response_help = "pets"
+	response_disarm = "gently pushes aside"
```

Comment:
```suggestion
```
Inherited.
And do you think the response_harm should be better as "splats" or "stamps on"?